python/ray/dashboard/client/src/api.ts

```diff
@@ -117,18 +117,10 @@ export type NodeInfoResponse = {
     };
     load_avg: [[number, number, number], [number, number, number]];
     net: [number, number]; // Sent and received network traffic in bytes / second
+    log_count?: { [pid: string]: number };
```

Comment:
Good question. The log entries by IP used to be a top level attribute, so the response looked like
```
nodeInfoResponse {
clients
log_counts
error_counts
}
``` 
where each client in clients represents one IP.

Now, the log_count and error_count exists on a per-client level, so each client has a `log_counts` and `error_counts` that corresponds to its IP. This simplifies the logic on the front-end a lot, since trying to refactor the log and error features to work with the `NodeFeatures` and `WorkerFeatures` arguments would have been difficult otherwise. 

This was changed on the backend accordingly.